includes/WatchedItem.php

```diff
@@ -20,18 +36,18 @@ class WatchedItem {
 	 * @return WatchedItem object
 	 */
 	public static function fromUserTitle( $user, $title ) {
-		$wl = new WatchedItem;
-		$wl->mUser = $user;
-		$wl->mTitle = $title;
-		$wl->id = $user->getId();
+		$watchedItem = new WatchedItem;
```

Comment:
Yea, it does. PHP let's you leave off the parenthesis if it's a no-arg constructor, but it does look weird. I'll add them.